README.md

```diff
@@ -92,6 +93,13 @@ module "bastion" {
 | bucket_name |  |
 | elb_ip |  |
 
+## OneLogin Sync
+
+Syncing users from OneLogin supported with onelogin_sync=true with the following requirements:
+1.  SSH Keys stored in a user custom attribute called 'sshPublickey'.
+2.  OneLogin credentials with Read perms stored in SSM Parameter Store parameters /bastion/onelogin_id and /bastion/onelogin_secret.
```

Comment:
When this goes live can you swap out the API key being used now named 'ericy_temp' to something less ambiguously named (maybe 'terraform_bastion' or something like that).

Reply:
Sure.  But let's keep this related to this PR.